docs/pages/2019_MARVEL_Psik_MaX/sections/setup.rst

```diff
@@ -138,11 +140,27 @@ terminal is the one we will actually use in this tutorial.
 
 .. note::
 
-   Our SSH configuration assumes that ``jupyter`` will serve the notebooks on port 8888. 
+   Our SSH configuration assumes that ``jupyter`` will serve the notebooks on port 8888.
    If you want to serve notebooks on different ports, you'll also need to adjust
    the SSH configuration.
 
 
+.. _setup_downloading_files:
+
+Downloading files
+-----------------
+
+Throughout this tutorial, you might encounter links through which you can download files, example and or solution scripts for example.
+Often the idea is to click on the link to download the file and put it in your working directory for the tutorial.
```

Comment:
These files should be downloaded to the environment/working directory you use to run the tutorial.